knees4bees/whats-cookin
# Feature/search favorites

--->  ((are you in the right branch??))
## What does this PR do?
 - Creates mechanism for user to search in favorites, not just in all recipes.
 - Adjusts placement and size of prominent text on list view, with title at top only on some views and search message now moved below search bar and above results.
 - Fixes bugs where html content would appear multiple times on repeated visits to same page (recipe detail view and user pantry)

## How should it be tested?
 - Open `index.html` and check all views. Note that some things are still not working right! Details on the Trello board.
 - (All tests are still passing except the ones that were broken earlier for `cookMeal`.)
